src/qibo/models/dbi/double_bracket.py

```diff
@@ -114,47 +151,31 @@ def backend(self):
         """Get Hamiltonian's backend."""
         return self.h0.backend
 
-    def hyperopt_step(
+    def least_squares(self, D: np.array):
+        """Least squares cost function."""
+        H = self.h.matrix
```

Comment:
```suggestion
        h = self.h.matrix
```
It's better to use lower case letter for variables :)